tests/drivers/spi/spi_loopback/prj_stm32f072b_disco.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_BOOT_BANNER=y
+CONFIG_BUILD_TIMESTAMP=y
+CONFIG_SYS_LOG=y
+CONFIG_SPI=y
+CONFIG_SPI_LEGACY_API=n
```

Comment:
It isn't needed. It defaults to n in arm/soc/st_stm32/common/Kconfig.defconfig.series when SPI and one SPI_x port are enabled.